narwhal/node/src/lib.rs

```diff
@@ -311,10 +320,7 @@ impl Node {
             store.certificate_store.clone(),
             &execution_state,
         )
-        .await?
-        .into_iter()
-        .sorted_by(|a, b| a.consensus_index.cmp(&b.consensus_index))
-        .collect::<Vec<ConsensusOutput>>();
```

Comment:
Github doesn't allow me to commend on a non-changed line, but let's change the `info!` message at line 328
https://github.com/MystenLabs/sui/pull/5900/files#diff-f6d9e991182542565a5ad10f44f1eeeb4a9536bcae12648c36be4a1f47a12ab0R328

to accurately capture the restored quantities. For example it could be something like:

```
Consensus output on its way to the executor was restored for {} leaders and {} certificates"
```